bundler/lib/dependabot/bundler.rb

```diff
@@ -12,12 +12,51 @@
 require "dependabot/bundler/version"
 
 require "dependabot/pull_request_creator/labeler"
+
+module Dependabot
+  module Bundler
+    PACKAGE_MANAGER = "bundler"
+    SUPPORTED_BUNDLER_VERSIONS = T.let(["2"].freeze, T::Array[String])
+    UNSUPPORTED_BUNDLER_VERSIONS = T.let([].freeze, T::Array[String])
+    DEPRECATED_BUNDLER_VERSIONS = T.let(["1"].freeze, T::Array[String])
+
+    class PackageManager < PackageManagerBase
```

Comment:
There's a PackageManager class under the NPM ecosystem that doesn't look much like this one. Are we going to reconcile the differences?

https://github.com/dependabot/dependabot-core/blob/8f11bbde6d10c28931c1d8fad60cb9de5891dcd1/npm_and_yarn/lib/dependabot/npm_and_yarn/package_manager.rb#L8